### PR TITLE
Update enroll-device-android-company-portal.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the embedded video to display

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3663

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/enroll-device-android-company-portal.md [hunk 1]
```diff
@@ -35,9 +35,10 @@ Enroll your personal or corporate-owned Android device with Intune Company Porta
 
 Company Portal supports devices running Android 8.0 and later, including devices secured by Samsung KNOX Standard 2.4 and later. To learn how to update your Android device, see [Check & update your Android version](https://support.google.com/android/answer/7680439).  
   
-</br>
+
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/enroll-device-android-company-portal.md:38]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: memdocs/intune/user-help/enroll-device-android-company-portal.md [hunk 2]
```diff
@@ -35,9 +35,10 @@ Enroll your personal or corporate-owned Android device with Intune Company Porta
...
 > [!VIDEO https://www.youtube.com/embed/k0Q_sGLSx6o]
 
+
 > [!NOTE]
 > Samsung Knox is a type of security that certain Samsung devices use for additional 
 > protection outside of what native Android provides. To check if you have a Samsung Knox device,> go to **Settings** > **About device**. If you don't see **Knox version** listed there, you have a native Android device.
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/enroll-device-android-company-portal.md:44]:
```suggestion
> protection outside of what native Android provides. To check if you have a Samsung Knox device, go to **Settings** > **About device**. If you don't see **Knox version** listed there, you have a native Android device.  
```

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Pull request merged**